vnext/Desktop.IntegrationTests/RNTesterIntegrationTests.cpp

```diff
@@ -16,6 +16,8 @@ std::wstring ToString<TestStatus>(const TestStatus &status) {
 
 } // namespace Microsoft::VisualStudio::CppUnitTestFramework
 
+// None of these tests are runnable
+#if 0
```

Comment:
Remove/deactivate TEST_CLASS attribute, but keep it compiling?

Reply:
Same for the other tests in the folder.

---
In reply to: [433406671](https://github.com/microsoft/react-native-windows/pull/5071#discussion_r433406671) [](ancestors = 433406671)